### PR TITLE
fix(core): android wrong background state + current value accessors

### PR DESCRIPTION
**Pull request by farfromrefug**
This fixes a bug with my recent addition of `backgroung/foreground` events.
It also adds getters `inBackground` and `suspended` in `application-common` to query the current value for both iOS/Android

### CODE REVIEW

**Review by NathanWalker — APPROVED** [review]

**Comment by farfromrefug** [comment]
@NathanWalker maybe the getters name should.be different.let me know if you think there would be better names

File: packages/core/application/index.android.ts [hunk 1]
```diff
@@ -48,8 +49,14 @@ export class AndroidApplication extends Observable implements AndroidApplication
...
 	private _systemAppearance: 'light' | 'dark';
-	public paused: boolean;
-	public backgrounded: boolean;
+
+	get paused() {
+		return suspended;
+	}
+	get backgrounded() {
+		return inBackground;
+	}
```
Comment (NathanWalker) [inline packages/core/application/index.android.ts:58]: @farfromrefug nice changes here, will definitely help. Let's go with this naming, I think will be more clear:
```
get isPaused() {
  return suspended;
}
get isInBackground() {
  return inBackground;
}
```

**Review by NathanWalker — CHANGES_REQUESTED** [review]

File: packages/core/application/index.android.ts (hunk 1 above)
Comment (farfromrefug) [reply packages/core/application/index.android.ts:58]: @NathanWalker i was more talking about the naming in application-common.ts https://github.com/Akylas/NativeScript/blob/backgroundevent_fix/packages/core/application/application-common.ts#L173=
The change you propose is a breaking change as it was already available. Are we sure we want that to change?
we could actually deprecate the android file one. Now people should use the common one as it works on iOS and android

Comment (NathanWalker) [reply packages/core/application/index.android.ts:58]: Oh! I see my bad. Let’s keep what you have then. Will include in next alpha.

Comment (NathanWalker) [reply packages/core/application/index.android.ts:58]: I agree with deprecating the old one and just make single consistent one from `Application.isPaused` for example - would behave same for all platforms. We can do that separately on the alpha branch itself.

**Pull request merged**